### PR TITLE
refactor(automation): Simplify docker compose files

### PR DESCRIPTION
**Pull request by Cliftonz**
This PR removes old and unused docker compose files and gives more flexibility to the base docker compose file by allowing the user to select the port of the infrastructure.

### CODE REVIEW

File: docker/local/deployment/docker-compose.monitoring.yml
Comment (SokratisVidros) [inline docker/local/deployment/docker-compose.monitoring.yml:1]: Should we merge the two docker-compose files into one? The `monitoring`, `local` suffixes are a bit misleading. We might prefer to use `docker-compose up SERVICE1 SERVICE2 ...` and target a single `docker-compose.yml`.

File: docker/local/development/docker-compose.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ services:
     environment:
       - SERVICES=s3
     ports:
-      - "4566:4566"
+      - "${DOCKER_LOCALSTACK_PORT}:4566"
```
Comment (SokratisVidros) [inline docker/local/development/docker-compose.yml]:
```suggestion
      - "${DOCKER_LOCALSTACK_PORT:-4566}:4566"
```

File: docker/local/development/docker-compose.yml [hunk 2]
```diff
@@ -21,7 +21,7 @@ services:
     image: mongo
     network_mode: bridge
     ports:
-      - "27017:27017"
+      - "${DOCKER_MONGO_PORT}:27017"
```
Comment (SokratisVidros) [inline docker/local/development/docker-compose.yml]:
```suggestion
      - "${DOCKER_MONGO_PORT:-27017}:27017"
```

File: docker/local/development/docker-compose.yml (hunk 1 above)
Comment (SokratisVidros) [reply docker/local/development/docker-compose.yml]: I am not a 100% sure if this works but it's a nice touch to add overridable, default values.

File: docker/local/development/docker-compose.yml [hunk 3]
```diff
@@ -33,7 +33,7 @@ services:
     image: redis
     network_mode: bridge
     ports:
-      - "6379:6379"
+      - "${DOCKER_REDIS_SERVICE_PORT}:6379"
```
Comment (SokratisVidros) [inline docker/local/development/docker-compose.yml]:
```suggestion
      - "${DOCKER_REDIS_SERVICE_PORT:-6379}:6379"
```

**Review by SokratisVidros — DISMISSED** [review]

File: docker/local/deployment/docker-compose.monitoring.yml
Comment (Cliftonz) [reply docker/local/deployment/docker-compose.monitoring.yml:1]: I think its fine to combine and remove all of the other files. I think there was alot of bloat in here and realistically we can make the sub systems profiles.

**Review by AliaksandrRyzhou — APPROVED** [review]

**Pull request merged**